gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -1029,7 +1029,13 @@ MM_VerboseHandlerOutput::outputCollectorHeapResizeInfo(MM_EnvironmentBase *env,
 const char *
 MM_VerboseHandlerOutput::getSubSpaceType(uintptr_t typeFlags)
 {
-	return "default";
+	const char *subSpaceType = NULL;
```

Comment:
This method is in base class. We need to override it in MM_VerboseHandlerOutputVLHGC (like MM_VerboseHandlerOutputStandard does) and leave this unchanged (or possibly make this pure virtual, if all other variants override it, but from what I quickly see, probably Realtime variant needs it).

Reply:
Yes good catch - refactoring this now. 
This function (`MM_VerboseHandlerOutput::getSubSpaceType`) will be reverted to it's original implementation